dietpi/dietpi-software

```diff
@@ -1178,6 +1178,16 @@ DietPi-Software will decrypt and use it for software installs. You can change it
 		aSOFTWARE_CATEGORY_INDEX[$software_id]=5
 		aSOFTWARE_ONLINEDOC_URL[$software_id]='p=10675#p10675'
 		aSOFTWARE_REQUIRES_JAVA_JRE_JDK[$software_id]=1
+    #------------------
+    software_id=181
+    
+    aSOFTWARE_NAME[$software_id]='Paper'
+		aSOFTWARE_DESC[$software_id]='A highly optimized server for Java Edition with plugins'
```

Comment:
What do you think about naming it `PaperMC` and "`Highly optimised Minecraft server with plugins, written in Java`"

Fits to the naming on their [website](https://papermc.io/) as well and at least gives a hint that it's a Minecraft server 😉.